### PR TITLE
BOM starter update

### PR DESCRIPTION
**Pull request by the-scrambler**

### CODE REVIEW

File: gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #Mon Oct 24 17:10:57 PDT 2022
 distributionBase=GRADLE_USER_HOME
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.4-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.5-bin.zip
```
Comment (osuleymanova) [inline gradle/wrapper/gradle-wrapper.properties:3]: 7.6- the latest

Comment (the-scrambler) [reply gradle/wrapper/gradle-wrapper.properties:3]: `7.5` is what was used in the last BOM update to the main branch. If I change it here we also have to change it there. @yenerm do you have thoughts on this?

Comment (osuleymanova) [reply gradle/wrapper/gradle-wrapper.properties:3]: ok, let's keep 7.5

**Review by osuleymanova — APPROVED** [review]

File: gradle/wrapper/gradle-wrapper.properties (hunk 1 above)
Comment (yenerm) [reply gradle/wrapper/gradle-wrapper.properties:3]: Let's go with 7.5

**Review by yenerm — APPROVED** [review]

**Review by yenerm — APPROVED** [review]

**Pull request merged**